popup1.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>KeepTabs List</title>
+    <link rel="stylesheet" type="text/css" href="popup.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+</head>
+<body>
+    <h1>My KeepTabs</h1>
+    <div class=table id="add">
+        <table>
+            <tr>
+                <th>Hotkey</th>
+                <th>URL</th>
+                <th><button type="button" class="save" value="SAVE"></th>
+            </tr>
+            <tr>
+            <!--use javascript to make a table with variable # rows-->
+                <form id="list">
+                    <td>
+                        <input type="text" id="hotkey">
+                    </td>
+                    <td>
+                        <input type="text" id="url">
+                    </td>
+                    <td>
+                        <input type="button"><i class="material-icons">expand_more</i>
+                    </td>
+                </form>
+            </tr>
+        </table>
+    </div>
+
+    <div id="domain">
+        <input type="button" value="Add new KeepTab">

```

Comment:
Not sure about this turning "KeepTab" into a noun. It can easily get confusing when you use the app's name for specific concepts within the app.